src/index.js

```diff
@@ -96,6 +139,13 @@ function removeTodo(index) {
   localStorage.setItem('todos', JSON.stringify(todos));
 }
 
+// edit todos on user input
+function editTodo(index, item) {
+  const todos = JSON.parse(localStorage.getItem('todos'));
+  todos[index].item = item;
+  localStorage.setItem('todos', JSON.stringify(todos));
+}
+
```

Comment:
- [ ] After editing an element and saving it, it would be great if the x icon can disappear and be replaced by the original icon. It would be great to implement this so as to improve the user interface.
- [ ] (OPTIONAL) Also if a user could press the enter key to save a task after editing, it would be great. Kindly add that too.

Reply:
Yes, sir I worked on that already